README.md

```diff
@@ -2,28 +2,36 @@
 
 UI for events from [Glutton](https://github.com/mushorg/glutton), streamed live from a development instance.
 
-## Development Requirements
-For backend development
+## Development Requirements -
 1. [Golang version > 1.17](https://go.dev/doc/install)
 2. [Node LTS version](https://nodejs.org/en/download/)
 
-### Steps for developement
+#### For windows system, some additional setup is needed (only in case of backend development)
```

Comment:
Should we point to the documentation [here ](https://github.com/mattn/go-sqlite3#windows) instead?

Reply:
We can sir, but I think we have to keep point 3 and 4 since the mentioned documentation contains `mingw32-make` and we might have to rename it but i am not sure of it. So its your call, whatever you suggest.

Reply:
@glaslos what do you suggest for this sir ?